x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -645,6 +648,47 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
                   data-test-subj="mlAnalyticsCreateJobFlyoutTrainingPercentSlider"
                 />
               </EuiFormRow>
+              {/* num_top_feature_importance_values */}
+              <EuiFormRow
+                label={i18n.translate(
+                  'xpack.ml.dataframe.analytics.create.numTopFeatureImportanceValuesLabel',
+                  {
+                    defaultMessage: 'Feature importance values',
+                  }
+                )}
+                helpText={i18n.translate(
+                  'xpack.ml.dataframe.analytics.create.numTopFeatureImportanceValuesHelpText',
+                  {
+                    defaultMessage:
+                      'Specify the maximum number of feature importance values per document to return.',
+                  }
+                )}
+                error={
+                  numTopFeatureImportanceValuesValid &&
+                  !sourceIndexNameEmpty && [
+                    i18n.translate(
+                      'xpack.ml.dataframe.analytics.create.numTopFeatureImportanceValuesErrorText',
+                      {
+                        defaultMessage: 'Invalid maximum number of feature importance values.',
+                      }
+                    ),
+                  ]
+                }
+              >
+                <EuiFieldNumber
```

Comment:
You need to add `min` and `step` properties in here, as otherwise you can enter e.g. `-1.23`. The Create button is disabled, but I think adding those props will prevent invalid input into the field.

![image](https://user-images.githubusercontent.com/7405507/78245051-a7808180-74de-11ea-9e88-4ec7770f1f8a.png)

Reply:
Fixed in b4e2b69.